citi-onboarding/iJunior-onepage
# Feature/frontend-contact

## What I did

- Created the front-end contact section

| Mockup | Real |
|--------|------|
| <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/123491248-ac05ee00-d5ec-11eb-943d-36b4f51268a3.png"> | <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/123491382-ea031200-d5ec-11eb-9997-4d0aac5a88ef.png"> |


## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/frontend-contact`
   1. `git pull origin feature/frontend-contact`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. In VScode open `/client/src/components and /views/Contacts`

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 2h50min

**Resolves:** # frontend of the Contact section
